### PR TITLE
Dt 4742 webpi

### PR DESCRIPTION
**Pull request by imehdihosseini**
# Description

Describe what this change achieves

## Issues Resolved

List any existing issues this PR resolves

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Installs/Configures the Microsoft Web Platform Installer (WebPI)'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '4.1.0'
+version          '4.1.1'
```
Comment (damacus) [inline metadata.rb:7]: Please revert this, our bot will bump the version for us 👍🏼

**Comment by damacus** [comment]
Did you maybe push this by accident to this repo? Looks like we have all of these changes (or most of them) on the mainline already.

**Comment by imehdihosseini** [comment]
Yes, sorry for disturbing you, I've push it to the wrong repo.

**Comment by damacus** [comment]
Hah, no problem! 

You might want to pull the latest version of the cookbook as it looks like your fork is out of date.

**Pull request closed without merging**